### ISSUE TITLE
Please disclose the product version in the main window of the program.

### ISSUE THREAD

**Issue by spgilmore** [wonderunit/storyboarder#1401]
When reporting a bug in the program, it was challenging to to determine the version of the product that I was using.  I had to open a new instance and found it on the launch screen.  I expected to find it in the HELP Menu or on the bottom margin in the "footer" area.

Even though I knew I likely had the latest version, The GitHub site didn't clearly indicate what the latest version was either.

Please add a menu item, HELP -> ABOUT which shows a dialogue wherein the version is displayed and, optionally, the build date.

**Comment by audionerd** [comment]
Yeah on macOS there is an "About Storyboarder" menu item, but it's not in the Windows version IIRC.

**Comment by squinonescolon** [comment]
The Linux version lacks one as well.
